raiden/blockchain_events_handler.py

```diff
@@ -35,6 +35,7 @@ def handle_tokennetwork_new(raiden, event: Event):
     block_number = data['block_number']
     token_network_address = args['token_network_address']
     token_address = typing.TokenAddress(args['token_address'])
+    block_hash = bytes(data['blockHash'])
```

Comment:
I think this should be done in the outer layer, respecting the sandwich encoding.